skyportal/handlers/api/spectrum.py

```diff
@@ -176,6 +177,100 @@ def delete(self, spectrum_id):
         return self.success()
 
 
+class ASCIIHandler(object):
```

Comment:
Dont need to subclass object in Python 3.x
```suggestion
class ASCIIHandler:
```

Reply:
fixed